src/test/kotlin/com/terraformation/backend/seedbank/search/SearchServiceTest.kt

```diff
@@ -1918,10 +1918,13 @@ class SearchServiceTest : DatabaseTest(), RunsAsUser {
       val viabilityTestResultsRow =
           ViabilityTestsRow(
               accessionId = AccessionId(1000),
-              testType = ViabilityTestType.Lab,
-              seedsSown = 15,
               remainingQuantity = BigDecimal.TEN,
-              remainingUnitsId = SeedQuantityUnits.Grams)
+              remainingUnitsId = SeedQuantityUnits.Grams,
+              seedsSown = 15,
```

Comment:
seedsTested? or maybe needs to be rebased off the other PR branch

Reply:
Admittedly this is a little confusing, but this is creating a `ViabilityTestsRow` which is a jOOQ-generated class whose field names match the database column names.

Reply:
Ok, I think I missed reviewing the context here. Thanks!